From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: SuperKali <hello@superkali.me>
Date: Tue, 21 Apr 2026 23:47:30 +0800
Subject: drm/bridge: simple: add the Chrontel CH7218A DP-to-HDMI bridge

The Chrontel CH7218A is a transparent Displayport to HDMI bridge.

Signed-off-by: Xilin Wu <sophon@radxa.com>

(cherry picked from commit 70c8c81198193a25ccb01368a831afae816d5408)
[SuperKali: backported to sc8280xp edge from radxa/kernel linux-7.0.11]
Signed-off-by: SuperKali <hello@superkali.me>
---
 drivers/gpu/drm/bridge/simple-bridge.c | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/drivers/gpu/drm/bridge/simple-bridge.c b/drivers/gpu/drm/bridge/simple-bridge.c
index 111111111111..222222222222 100644
--- a/drivers/gpu/drm/bridge/simple-bridge.c
+++ b/drivers/gpu/drm/bridge/simple-bridge.c
@@ -275,6 +275,11 @@ static const struct of_device_id simple_bridge_match[] = {
 		.data = &(const struct simple_bridge_info) {
 			.connector_type = DRM_MODE_CONNECTOR_HDMIA,
 		},
+	}, {
+		.compatible = "chrontel,ch7218a",
+		.data = &(const struct simple_bridge_info) {
+			.connector_type = DRM_MODE_CONNECTOR_HDMIA,
+		},
 	}, {
 		.compatible = "parade,ps185hdm",
 		.data = &(const struct simple_bridge_info) {
-- 
Armbian

